### DISCUSSION TITLE
Error pokedex live coding

### DISCUSSION THREAD

**Discussion by GabrielVillagran** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#439] [General]
Hola, que tal compañeros tengo una duda que sinceramente no sé bien como explicar jeje... cuando pruebo mi código con la extensión live server todo funciona a la perfección como en esta imagén 
![image](https://user-images.githubusercontent.com/44887537/158746201-52a3d9e0-710e-4279-942f-3a87de7f59ff.png)

pero cuando abro mi archivo html directo en el navegador me arroja el siguiente error:
![image](https://user-images.githubusercontent.com/44887537/158746422-021c4dbd-c374-40bb-a6fd-7d215af6c278.png)

A que se podra deber el error, me parece extraño

**Comment by alejandronjj** [comment]
Ahí dice que no encuentra el archivo de javascript, por lo tanto no conoce la función fetchpokemon. Necesitaría ver la parte del código donde llamas el script pokedex.js. Checa que el nombre y la ruta sean correctos.
